pkg/config/lpe_unix.go

```diff
@@ -0,0 +1,56 @@
+//go:build aix || darwin || dragonfly || freebsd || linux || netbsd || openbsd || solaris || zos
```

Comment:
just use `!windows` this is waht we already in the other code paths.

Reply:
> just use !windows this is waht we already in the other code paths.
Original implementation provided special version for Plan9 systems. If this is never a case for this project, then it will work.